### PR TITLE
User api

### PR DESCRIPTION
**Pull request by sumit-vishwakarma**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -73,15 +73,16 @@ <h3 class="modal-title">Add Meeting</h3>
               <div class="form-group">
                 <label for="roomType" class="py-2">Room</label>
                 <select class="form-control" id="roomType" required>
-                  <option value="A1">A1</option>
-                  <option value="A2">A2</option>
+                  <option value="1">A1</option>
+                  <option value="2">A2</option>
```
Comment (govind-sharma-4112) [inline index.html:77]: create common component for meeting

File: js/dashboard.js [hunk 2]
```diff
@@ -50,142 +42,139 @@ const dashboardScreen = (meetings) => {
...
+    !meeting_date &&
+    !start_time &&
+    !end_time
+  ) {
+    alert("Please, Fill all Field");
+  } else {
+    console.log(meeting_date);
+    axios
+      .post("http://localhost:8080/meeting/add", {
+        meeting_name,
```
Comment (govind-sharma-4112) [inline js/dashboard.js]: create a separate variable and pass in API param

File: js/dashboard.js [hunk 3]
```diff
@@ -50,142 +42,139 @@ const dashboardScreen = (meetings) => {
...
-//             alert("Slots added Succesfully")
-//         } else {
-//             alert("Someting Went Wrong");
-//         }
-        
-//         location.reload()
-//     }
-// }
+const saveId = (id) => {
+  localStorage.setItem("updateMeetingId", id);
```
Comment (govind-sharma-4112) [inline js/dashboard.js]: add variable to store id remove asyncstorage

File: js/dashboard.js [hunk 4]
```diff
@@ -50,142 +42,139 @@ const dashboardScreen = (meetings) => {
...
+    !start_time &&
+    !end_time
+  ) {
+    alert("Please, Fill all Field");
+  } else {
+    const id = localStorage.getItem("updateMeetingId");
+    axios
+      .put(`http://localhost:8080/meeting/update/${id}`, {
+        meeting_name,
+        fk_room_id,
```
Comment (govind-sharma-4112) [inline js/dashboard.js]: create variable and assign the variable in API param

File: js/dashboard.js (hunk 2 above)
Comment (sumit-vishwakarma) [reply js/dashboard.js]: done

File: js/dashboard.js (hunk 4 above)
Comment (sumit-vishwakarma) [reply js/dashboard.js]: done

File: index.html (hunk 1 above)
Comment (sumit-vishwakarma) [reply index.html:77]: done

**Pull request merged**